frontend/src/scss/PrimeStyles.scss

```diff
@@ -206,3 +208,18 @@
 .Toastify__toast {
   padding: 3px;
 }
+
+// very basic required-field indicator and validator
+// can't use outline because USDS style already overloads it
```

Comment:
There needs to be a textual indicator, it can't just be color because a screen reader won't understand it. USWDS guidelines say to mark optional fields. 

> Usability guidance : Don’t call out required fields. Call out optional fields instead. Users can infer that non-optional fields are required. -  https://designsystem.digital.gov/components/form-templates/

Reply:
Yeah, this is an outstanding issue for the whole project.  Do you think it is enough to put the `(required)` in the column label?